release-calendar/src/client/components/App.tsx

```diff
@@ -13,10 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import * as React from 'react';
+import {Calendar} from './Calendar';
+import {FAKEEVENTS} from './fakeData';
 import {Header} from './Header';
 
 export const App = (): JSX.Element => {
-  return <Header title='AMP Release Calendar' />;
+  return (
+    <div>
+      <Header title='AMP Release Calendar' />
```

Comment:
Can this be styled as well? Right now it's a different font than the calendar, off to the left.

Reply:
I've been waiting to figure out font until I've add the material design. My color palette and other design choices like curves or right angles are also up in the air until then.

Reply:
Ok makes sense!